pkgs/servers/sql/postgresql/libpq.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv
+, lib
+, openssl
+, zlib
+, libkrb5
+, postgresql
+, perl
+, bison
+, flex
+, tzdata
+, gssSupport ? with stdenv.hostPlatform; !isWindows && !isStatic
+}:
+
+stdenv.mkDerivation {
+  pname = "libpq";
+  version = postgresql.version;
+  src = postgresql.src;
```

Comment:
Any concerns about referencing the `postgresql` derivation here?

Reply:
If there's no source tarball dedicated for `libpq`, then there's not much we can do about this anyways because the full src tarball would still be in the build closure.